appveyor.yml

```diff
@@ -91,16 +91,20 @@ for:
   # scripts to run after tests
   after_test:
   - ps: |
+      & pwd
       & Setup\BuildInstallers.cmd
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
       & Setup\Set-Portable.ps1
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
       & Setup\MakePortableArchive.cmd Release $env:APPVEYOR_BUILD_VERSION
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
```

Comment:
@vbjay removing this we risk publishing broken things...

Reply:
That was so tree is always generated.  If it fails, tree is very useful when it can't find a file.  We could use a variable and put one of those lines after tree except use the variable.  

```powershell
if ($LastExitCode -ne 0) {doExit=$true}
....
if($doExit-eq $true){ $host.SetShouldExit($LastExitCode) }
```